app/OParl/OParlApiManager.php

```diff
@@ -52,21 +46,66 @@ public static function organization(string $organizationId)
         return new Organization($data);
     }
 
-    public static function organizations(Collection $organizationIds = null)
+    public static function organizations($page = null, Collection $organizationIds = null)
     {
         if ($organizationIds) {
             return $organizationIds->map(function ($id) {
                 return self::organization($id);
             });
         }
 
-        $data = resolve(\App\Contracts\OParlApi::class)->organizations();
+        $data = resolve(\App\Contracts\OParlApi::class)->organizations($page);
 
-        $oganizations = collect($data->get('data'))->map(function ($organizationData) {
-            return new Organization($organizationData);
-        });
+        return [$data['data'],  self::extractPages($data)];
+    }
+
+    public static function locations($page = null)
+    {
+        $data = resolve(\App\Contracts\OParlApi::class)->locations($page);
+
+        return [$data['data'],  self::extractPages($data)];
+    }
+
+    public static function papers($page = null)
+    {
+        $data = resolve(\App\Contracts\OParlApi::class)->papers($page);
+
+        return [$data['data'],  self::extractPages($data)];
```

Comment:
two things here:
1. you're almost always using: ` return [$data['data'],  self::extractPages($data)];` so a simple function wich gets data and returns this would leave you with more dynamic and an easier to rewrite code if something changes on how you like to return data from the API manager.

2. i'm not sure if a unnamed Array is so ideal as return. i know it's cool to use with list. but wouldnt a value object which concsts of data and pagination may be better? or at least a named array with `['data' => ..., 'pages' => ...]`